src/main/osd/osd_warnings.c

```diff
@@ -76,6 +76,35 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
     static armingDisableFlags_e armingDisabledDisplayFlag = 0;
 
     warningText[0] = '\0';
+
+#ifdef USE_GPS_RESCUE
+    if (osdWarnGetState(OSD_WARNING_GPS_RESCUE_UNAVAILABLE)
+      && ARMING_FLAG(ARMED)
+      && gpsRescueIsConfigured()
```

Comment:
opening `(` not closed with a  `)` and non-existent opening`{`

Reply:
Thank you @nerdCopter I got a bit confused there